### PR TITLE
div. Bugfixes

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: grails-app/domain/de/laser/Task.groovy [hunk 1]
```diff
@@ -143,4 +143,27 @@ class Task {
         }
         displayArgs
     }
+
+    String geObjectName() {
```
Comment (klodav) [inline grails-app/domain/de/laser/Task.groovy:147]: typo -> getObjectName

File: grails-app/domain/de/laser/Task.groovy [hunk 2]
```diff
@@ -143,4 +143,27 @@ class Task {
...
+        else if (pkg) {
+            name = pkg.name
+        }
+        else if (subscription) {
+            name = subscription.name
+        }
+        else if (surveyConfig) {
+            name = surveyConfig.surveyInfo.name
+        }
+        else {
```
Comment (klodav) [inline grails-app/domain/de/laser/Task.groovy:164]: redundant

File: grails-app/domain/de/laser/Task.groovy (hunk 1 above)
Comment (Moetez16) [reply grails-app/domain/de/laser/Task.groovy]: Done!

File: grails-app/domain/de/laser/Task.groovy (hunk 2 above)
Comment (Moetez16) [reply grails-app/domain/de/laser/Task.groovy]: Done!

**Review by klodav — APPROVED** [review]

**Pull request merged**